docs/workflow/building/coreclr/cross-building.md

```diff
@@ -134,13 +136,13 @@ The following instructions assume you are on a Linux machine such as Ubuntu 14.0
 To build System.Private.CoreLib for Linux, run the following command:
 
 ```
-    lgs@ubuntu ~/git/coreclr/ $ build.sh arm debug verbose
+    lgs@ubuntu ~/git/runtime/ $ ./build.sh --subsetCategory CoreClr --subset CoreLib-NativeCoreLib --arch arm -c debug -v verbose
```

Comment:
You can just use a single - everywhere. Also lowercasing everything works as well.

Reply:
Making these consistent can be a separate PR.

- The existing docs use a mix of `-` and `--`. 
- `-subset help` lists the names as CamelCased. It makes sense for the docs to match it.